### PR TITLE
[INLONG-6104][Manager] Support getting backup info in getSortSource

### LINKED ISSUE #6104: [Feature][Manager] Support getting backup info in getSortSource
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
- Fixes #6104

### Motivation

1. To unify the metadata management of sort-standalone and sort-flink
2. Use cursor to reduce the database load when select all entries of a specify table.

### Modifications

1. Parse **_backup_clusterTag_**  in inlong_group_ext table
2. Parse  **_backup_mqResource_** in inlong_stream_ext table
3. Use cursor to select all infos 

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is already covered by existing tests, such as:
- [ ] SortClusterServiceImpl

### Documentation

  - Does this pull request introduce a new feature?  no

### CODE REVIEW

**Review by luchunliang — APPROVED** [review]
LGTM

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/sort/SortServiceImplTest.java [hunk 1]
```diff
@@ -195,20 +216,50 @@ private void prepareDataNode(String taskName) {
...
-        entity.setModifyTime(new Date());
-        entity.setIsDeleted(InlongConstants.UN_DELETED);
-        entity.setVersion(InlongConstants.INITIAL_VERSION);
-        inlongGroupEntityMapper.insert(entity);
+        InlongPulsarRequest request = new InlongPulsarRequest();
+        request.setInlongGroupId(groupId);
+        request.setMqResource("test namespace");
+        request.setInlongClusterTag(TEST_TAG);
+        request.setVersion(InlongConstants.INITIAL_VERSION);
+        request.setName("test group name");
```
Comment (healchow) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/sort/SortServiceImplTest.java:224]: Suggest adding `-` or `_` between words for name, backup topic, etc.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortSourceServiceImpl.java [hunk 2]
```diff
@@ -177,6 +194,156 @@ public SortSourceConfigResponse getSourceConfig(
...
+    }
+
+    private Map<String, CacheZone> parsePulsarCacheZones(List<InlongGroupTopicInfo> groupTopicInfos) {
+        Map<String, CacheZone> cacheZoneMap = new HashMap<>();
+        groupTopicInfos.forEach(info -> this.parseCacheZones(info, cacheZoneMap));
+        return cacheZoneMap;
+    }
+
+    private void parseCacheZones(InlongGroupTopicInfo info, Map<String, CacheZone> cacheZoneMap) {
+        switch (info.getMqType().toUpperCase()) {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortSourceServiceImpl.java:261]: No need to upper case, it is an error if it was not matched.

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/sort/SortServiceImplTest.java (hunk 1 above)
Comment (vernedeng) [reply inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/sort/SortServiceImplTest.java:224]: fixed, thx

File: inlong-manager/manager-dao/src/main/resources/mappers/StreamSinkEntityMapper.xml [hunk 3]
```diff
@@ -350,6 +350,13 @@
             </if>
         </where>
     </select>
+    <select id="selectAllStreamSinks" resultType="org.apache.inlong.manager.dao.entity.StreamSinkEntity">
+        select
+        <include refid="Base_Column_List"/>
+        from stream_sink
+        where is_deleted = 0
+        group by inlong_cluster_name, sort_task_name, sort_consumer_group, sink_type, data_node_name
```
[the remark this answers is not in the document]
Comment (vernedeng) [reply inlong-manager/manager-dao/src/main/resources/mappers/StreamSinkEntityMapper.xml:358]: fixed, thx

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortSourceServiceImpl.java (hunk 2 above)
Comment (vernedeng) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortSourceServiceImpl.java:261]: fixed, thx

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongStreamEntityMapper.java [hunk 4]
```diff
@@ -71,4 +73,5 @@ int updateStatusByIdentifier(@Param("groupId") String groupId, @Param("streamId"
      */
     int deleteByInlongGroupIds(@Param("groupIdList") List<String> groupIdList);
 
+    Cursor<SortSourceStreamInfo> selectAllStreams();
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongStreamEntityMapper.java:76]: Did this Cursor need `@Options`?

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/standalone/SortSourceStreamSinkInfo.java [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    String extParams;
+    Map<String, String> extParamsMap;
+
+    public Map<String, String> getExtParamsMap() {
+        if (extParamsMap != null) {
+            return extParamsMap;
+        }
+        if (StringUtils.isNotBlank(extParams)) {
+            try {
+                Gson gson = new Gson();
```
Comment (healchow) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/standalone/SortSourceStreamSinkInfo.java:46]: Suggest using a global instance.

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongStreamEntityMapper.java (hunk 4 above)
Comment (vernedeng) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongStreamEntityMapper.java]: Yes, I forgot it.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/SortConfigLoader.java [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+import org.apache.inlong.manager.dao.entity.InlongGroupExtEntity;
+import org.apache.inlong.manager.dao.entity.InlongStreamExtEntity;
+import org.apache.inlong.manager.pojo.sort.standalone.SortSourceClusterInfo;
+import org.apache.inlong.manager.pojo.sort.standalone.SortSourceGroupInfo;
+import org.apache.inlong.manager.pojo.sort.standalone.SortSourceStreamInfo;
+import org.apache.inlong.manager.pojo.sort.standalone.SortSourceStreamSinkInfo;
+
+import java.util.List;
+
+public interface SortConfigLoader {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/SortConfigLoader.java:29]: Please add Java docs for classes, interfaces, and public methods.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/standalone/SortSourceStreamSinkInfo.java (hunk 5 above)
Comment (vernedeng) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/standalone/SortSourceStreamSinkInfo.java]: fixed, thx

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/SortConfigLoader.java (hunk 6 above)
Comment (vernedeng) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/SortConfigLoader.java:29]: fixed, thx

**Review by healchow — APPROVED** [review]

**Review by ciscozhou — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**